addons/func_godot/src/map/func_godot_map.gd

```diff
@@ -1007,6 +1007,11 @@ func apply_properties_and_finish() -> void:
 		
 		if 'func_godot_properties' in entity_node:
 			entity_node.func_godot_properties = properties
+
+		if override_node_properties:
+			for i in properties:
```

Comment:
Should replace `i` with `property`, to match earlier conventions and for clarity.